command-expansion-server/src/app.ts

```diff
@@ -433,6 +433,44 @@ export interface SeqBuilder {
   ): Sequence;
 }
 
+//generate a sequence JSON from a sequence file
+app.post('/get-sequence-edsl', async (req, res, next) => {
+  const commandDictionaryID = req.body.input.commandDictionaryID as number;
+  const edslBody = req.body.input.edslBody as string;
+
+  let commandTypes;
+  try {
+    const context: Context = res.locals['context'];
+    commandTypes = await context.commandTypescriptDataLoader.load({ dictionaryId: commandDictionaryID });
+  } catch (e) {
+    res.status(200).json({
+      message: 'Error loading command dictionary',
+      cause: (e as Error).message,
+    });
+    return next();
+  }
+
+  const result = await ((await piscina.run(
+    {
+      edslBody,
+      commandTypes,
+    },
+    { name: 'executeEDSL' },
+  )) as ReturnType<typeof executeEDSL>);
+
+  const sequenceJson = result.sequenceJson;
+  if (sequenceJson != null) {
```

Comment:
I'm going to claim that errors should be the differentiating check here instead of sequenceJson existence. Err on the side of aggressive failure rather than silent success and suppression of errors.